.taskcluster.yml

```diff
@@ -1,40 +1,47 @@
 # The version is always required
-version: 0
-allowPullRequests: public
+version: 1
+policy:
+  pullRequest: public
 # Top level metadata is always required
-metadata:
-  name: "Taskcluster Generic Worker Tests"
-  description: "These tests should ensure that any new commits against generic worker codebase are tested across all supported worker types in gekco."
-  owner: "{{ event.head.user.email }}" # the user who sent the pr/push e-mail will be inserted here
-  source: "{{ event.head.repo.url }}"  # the repo where the pr came from will be inserted here
+#metadata:
+#  name: Taskcluster Generic Worker Tests
+#  description: These tests should ensure that any new commits against generic worker codebase are tested across all supported worker types in gekco.
+#  owner: ${owner} # the user who sent the pr/push e-mail will be inserted here
+#  source: ${event.head.repo.url}  # the repo where the pr came from will be inserted here
 tasks:
-
-
-  ##########################################################
-  ######### Windows Server 2012 R2 Multiuser Build #########
-  ##########################################################
-
-  - provisionerId: pmoore-test
-    workerType: win2012r2-cu
-    metadata:
-      name: "Build/test 64 bit generic-worker (multiuser engine) on Windows Server 2012 R2"
-      description: "This builds and tests the 64 bit Windows version of generic-worker (multiuser engine) on Windows Server 2012 R2"
-      owner: "{{ event.head.user.email }}" # the user who sent the pr/push e-mail will be inserted here
-      source: "{{ event.head.repo.url }}"  # the repo where the pr came from will be inserted here
-    extra:
-      github:
-        # Events that will trigger this task
-        events:
-          - pull_request.opened
-          - pull_request.synchronize
-          - push
-    scopes:
-      - generic-worker:cache:generic-worker-checkout
-      - secrets:get:repo:github.com/taskcluster/generic-worker
-    payload:
-      features:
-        taskclusterProxy: true
-      maxRunTime: 3600
+  $let:
+    owner: "taskcluster-internal@mozilla.com" # the user who sent the pr/push e-mail will be inserted here
+    repo:
+      $if: 'tasks_for == "github-pull_request" && event["action"] in ["opened", "synchronize"]'
+      then:
+        url: ${event.pull_request.head.repo.clone_url}
```

Comment:
Instead of having variables in an object structure (`repo.url`), we can just declare variables with names like `repo_url` or something similar, or even just `repository`. This might take care of `InterpreterError at template.tasks.ref: object has no property ref`